app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
### [OPTIONAL]

- It will be great to use `[Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html)`  after_save to call your update_likes_counter method every time a new post has been saved to the database. This will be a much better implementation.